libs/core/include/core/feature_flags.hpp

```diff
@@ -31,6 +31,8 @@ namespace core {
 class FeatureFlags
 {
 public:
+  constexpr static char const *const MAIN_CHAIN_BLOOM_FILTER = "main_chain_bloom_filter";
```

Comment:
Pedantic: I believe the final `const` is redundant because of the `constexpr` also as for consistency normally these are done: `static constexpr char const *`

Reply:
But more const == more better...! Fine.